### ISSUE TITLE
[1.20.1 forge] LDLib tanks performance in my modpack 

### ISSUE THREAD

**Issue by Aabyonis** [Low-Drag-MC/LDLib-MultiLoader#44]
![2024-10-03_04 12 04](https://github.com/user-attachments/assets/220a0145-f551-4ed7-933b-85096656ec3d)
![2024-10-03_04 08 49](https://github.com/user-attachments/assets/3fe68eb2-06a2-48b9-a0a9-06b40764d4cd)
The library itself without the mod tanks my game performance quite abit i thought this maybe of concern 

here is the log when the game was having a performance loss:
[ldlib performance.log](https://github.com/user-attachments/files/17237631/ldlib.performance.log)

**Comment by Yefancy** [comment]
could you help me test only ldlib isntalled and only forge loader witout any mods?

**Comment by Yefancy** [comment]
I cant reproduce it. Im worried about being incompatible with other mods

**Comment by Yefancy** [comment]
I have tested with embeddium and ldlib

**Comment by Aabyonis** [comment]
> could you help me test only ldlib isntalled and only forge loader witout any mods?

Yeah i just did, it doesnt seem to drop when its only LDLib and forge loader

**Comment by Aabyonis** [comment]
i suspect this might be a mod conflict, but might be very hard to tell without a binary search of all mods

**Comment by Yefancy** [comment]
> i suspect this might be a mod conflict, but might be very hard to tell without a binary search of all mods

true, it would be appreciated if you could help me figure it out. Its fine if you dont have time, take your time.

**Comment by Aabyonis** [comment]
> > i suspect this might be a mod conflict, but might be very hard to tell without a binary search of all mods
> 
> true, it would be appreciated if you could help me figure it out. Its fine if you dont have time, take your time.

i managed to figure it out, it was caused by the mod "wakes" that was running on top of sinytra connector on forge, i thought sinytra would be the culprit but it wasnt, it was the wakes mod for some reason, if you would like to reproduce install sinytra connector and wakes mod on 1.20.1 forge 47.3.0

**Comment by Aabyonis** [comment]
> > > i suspect this might be a mod conflict, but might be very hard to tell without a binary search of all mods
> > 
> > 
> > true, it would be appreciated if you could help me figure it out. Its fine if you dont have time, take your time.
> 
> i managed to figure it out, it was caused by the mod "wakes" that was running on top of sinytra connector on forge, i thought sinytra would be the culprit but it wasnt, it was the wakes mod for some reason, if you would like to reproduce install sinytra connector and wakes mod on 1.20.1 forge 47.3.0

if you could look into why that happens it would be much appreciated

**Comment by Tcat2000** [comment]
Hi, a little late, but I assume you tested with just wakes, just seems like it could be a laggy kind of mod, especially over connector. That and I don't see why this would be a conflict.
